internal/cmdrender/runtime/container.go

```diff
@@ -103,3 +126,34 @@ func (f *ContainerFn) getDockerCmd() (*exec.Cmd, context.CancelFunc) {
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	return exec.CommandContext(ctx, path, args...), cancel
 }
+
+// prepareImage will check local images and pull it if it doesn't
+// exist.
+func (f *ContainerFn) prepareImage() error {
+	// check image existence
+	path := "docker"
```

Comment:
don't hardcode the binary in multiple places.

Reply:
done